.github/workflows/publish-shared-sdk.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: publish-shared-sdk
+
+on: workflow_dispatch
+
+jobs:
+  publish-shared-sdk:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout hcp-sdk-go
+        uses: actions/checkout@v2
+        with:
+          ref: main
+          path: hcp-sdk-go
+
+      - name: Checkout cloud-api
+        uses: actions/checkout@v2
+        with:
+          repository: hashicorp/cloud-api
+          ref: master
```

Comment:
I'm guessing this is fine, but I did want to understand how this is working. Is there a reason we use `master` here, but for the services we're using `auto-update-service-specs`?

Reply:
Can't remember if I did this on purpose or not 😂 but I think we'll want a separate branch